src/jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.tsx

```diff
@@ -85,17 +87,15 @@ export const DesktopArticleVideoPlayerContent: React.FC<DesktopArticleVideoPlaye
 						className={clsx({
 							[styles.desktopArticleVideoWrapper]: !isScrollPlayer,
 							[styles.desktopArticleVideoWrapperScrollPlayer]: isScrollPlayer,
+							[styles.desktopArticleVideoWrapperScrollPlayerCustomized]: isScrollPlayer && shouldCustomizeBehavior,
```

Comment:
Not sure what's the purpose of it 🤔 There should be no scroll experience for the partners player so I don't think we need to add any extra class.

Reply:
When all the styling is removed, the player's display becomes faulty and flickers. This only addresses the issue of the player being displayed off the screen.

Reply:
Okay, I initially thought that the request was only to fix the current customization of the floating player experience, but now I understand that the request is to completely remove it.